### PR TITLE
New package: Soniczac7.DownloadManager version 5.1.0.0

### PR DESCRIPTION
**Pull request by Lilyy2565**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/174488)

### CODE REVIEW

**Comment by Soniczac7** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 896.)

File: manifests/s/Soniczac7/DownloadManager/5.1.0.0/Soniczac7.DownloadManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageIdentifier: Soniczac7.DownloadManager
+PackageVersion: 5.1.0.0
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: DownloadManagerInstaller.exe
+InstallerSwitches:
+  Silent: --install
+  SilentWithProgress: --install
+Installers:
```
Comment (stephengillie) [inline manifests/s/Soniczac7/DownloadManager/5.1.0.0/Soniczac7.DownloadManager.installer.yaml:13]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with:
image
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> MSI (s) (6C:B8) [11:50:16:999]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (6C:B8) [11:50:16:999]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (6C:B8) [11:50:16:999]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (6C:B8) [11:50:16:999]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (6C:B8) [11:50:16:999]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.
> MSI (s) (6C:B8) [11:50:18:561]: Executing op: ComponentRegister(ComponentId={8C08EE04-B7BF-5CFB-9D47-C0E09E7A33C2},KeyPath=22*85*\SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules\C:\Program Files\dotnet\shared\Microsoft.NETCore.App\6.0.33\mscordaccore.dll,State=3,,Disk=1,SharedDllRefCount=0,BinaryType=1)
> MSI (s) (6C:B8) [11:50:20:296]: Executing op: RegOpenKey(Root=-2147483646,Key=SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules,,BinaryType=1,,)
> Property(S): ErrorDialog = ErrorDlg
> Property(S): OutOfDiskSpace = 0
> Property(S): OutOfNoRbDiskSpace = 0
> Property(S): PrimaryVolumeSpaceAvailable = 0
> Property(S): PrimaryVolumeSpaceRequired = 0
> Property(S): PrimaryVolumeSpaceRemaining = 0
> MSI (s) (6C:C8) [11:50:24:061]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (6C:C8) [11:50:24:061]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (6C:C8) [11:50:24:061]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (6C:C8) [11:50:24:061]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (6C:C8) [11:50:24:061]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.
> MSI (s) (6C:C0) [11:50:24:889]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (6C:C0) [11:50:24:889]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (6C:C0) [11:50:24:889]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (6C:C0) [11:50:24:889]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (6C:C0) [11:50:24:889]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.
> MSI (s) (6C:84) [11:50:26:452]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (6C:84) [11:50:26:452]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (6C:84) [11:50:26:452]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (6C:84) [11:50:26:452]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (6C:84) [11:50:26:452]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.

(Automated response - build 896.)

**Comment by Soniczac7** [comment]
Hi @stephengillie,
It was an issue with the installer, I have patched the issue and updated the release and it should work as expected now.

**Comment by stephengillie** [comment]
Verify:
image
image
The installer doesn't write a version number to the Registry.

**Review by stephengillie — APPROVED** [review]

**Comment by Soniczac7** [comment]
For this version it is intended however will be changed to have the version number available in the next release.

**Comment by stephengillie** [comment]
The main concern is around version matching. The package manager offers upgrades by matching data in the Registry against a downloaded copy of the manifests in this repo. If other manifests for this `PackageIdentifier` have this data and this manifest does not, I'm not sure of the outcome, but I'm guessing this version alone might have an unexpected situation. But that issue can be solved in the future, and this manifest should stay, at least for now.

**Pull request merged**